apps/molgenis-components/lib/main.js

```diff
@@ -47,12 +53,30 @@ import ObjectDisplay from "../src/components/tables/cellTypes/ObjectDisplay.vue"
 import StringDisplay from "../src/components/tables/cellTypes/StringDisplay.vue";
 import TextDisplay from "../src/components/tables/cellTypes/TextDisplay.vue";
 
-import SignInForm from "../src/components/organisms/SignInForm.vue"
-
+import MolgenisSignIn from "../src/components/layout/MolgenisSignIn.vue";
+import ShowMore from "../src/components/layout/ShowMore.vue";
 import Client from "../src/client/client.js";
 
+import EmailDisplay from "../src/components/tables/cellTypes/EmailDisplay.vue";
+import HyperlinkDisplay from "../src/components/tables/cellTypes/HyperlinkDisplay.vue";
+import InputHeading from "../src/components/forms/InputHeading.vue";
+import LayoutForm from "../src/components/layout/LayoutForm.vue";
+import MessageWarning from "../src/components/forms/MessageWarning.vue";
+import MolgenisMenu from "../src/components/layout/MolgenisMenu.vue";
+
+import LayoutCard from "../src/components/layout/LayoutCard.vue";
```

Comment:
please use folder order (see like the other components ) else its hard to keep track of inclusion.

Reply:
done